docker/checkout.bash

```diff
@@ -74,3 +78,10 @@ cd ${dir}/src/ros2_tracing
 git sparse-checkout init --cone
 git sparse-checkout set tracetools
 cd ${dir}/src
+
+cd ${dir}
+
+# Install dependencies
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+sudo chmod +x ${SCRIPT_DIR}/install_dependencies.sh
+${SCRIPT_DIR}/install_dependencies.sh -b $BRANCH -r $dir
```

Comment:
NIt: Can we wrap this with `if [[ -z "$PACKAGES" ]]; then` so that it is a little bit more obvious if we are running this or not.  Although I know that if clause is still inside.